test/e2e/tests/metrics/metametrics-persistence.spec.js

```diff
@@ -23,14 +18,6 @@ describe('MetaMetrics ID persistence', function () {
             participateInMetaMetrics: true,
           })
           .build(),
-        localNodeOptions: generateGanacheOptions({
-          accounts: [
-            {
-              secretKey: defaultGanacheOptions.accounts[0].secretKey,
-              balance: initialBalanceInHex,
```

Comment:
no reason at all to start with a random balance, as this is not checked in the test, which we just check metric related topics, so we can remove it safely